draft-ion.adoc

```diff
@@ -208,79 +200,176 @@ represented as follows:
+  ]
+}
+----
+
+The above example has 2 Ion Links in its `items` array.  Each of these 2 links are implicitly assigned the {uri-linkRelType-item}[`item`] link relation type.
+
+==== Explicit Link Relation Types
+
+An Ion link MAY have one or more explicitly defined link relation types declared as an array of strings assigned to the link Meta Object's `rel` member.
+
+A non-normative example of a link with an explicitly defined link relation type:
+
+.Example Ion Link with explicit link relation type:
+[source,json]
+----
+{ "meta": { "href": "https://cdn.example.com/example.ico", "rel": ["icon"] } }
+----
+
+The above example has an explicit link relation type of http://www.w3.org/TR/html5/links.html#rel-icon[`icon`].
+
+More than one explicit link relation type MAY be defined in the `rel` array.
+
+Ion parsers MUST NOT recognize any non-string value in the `rel` array as a link relation type.
+
+Ion parsers MUST treat null or empty `rel` values as if the `rel` property was not defined at all.
+
+== Registered Metadata Member Names
+
+The following member names are registered in the IANA Ion Metadata member registry
+defined in [TBD].  None of the members defined below are intended to be mandatory
 in all cases, but rather, provide an initial set likely to be useful for common
 use cases.
 
-==== `href`
+=== `href`
 
-A resource's {uri-IRIs}[IRI] (Internationalized Resource Identifier).
+The `href` (hypermedia reference) member identifies the target {uri-IRIs}[IRI] (Internationalized Resource Identifier) location of the resource. The `href` value is a case-sensitive {uri-IRIs}[IRI] value.
 
-A _Link_ MUST contain an `href` field.  A _Link_ `href` IRI is the location of
- the linked resource, called the "target IRI".
-
-A _Meta Object_ MAY contain an OPTIONAL `href` field.  If present, the
-_Meta Object_ `href` field IRI is the location of the parent resource, called
-the "context IRI". It is RECOMMENDED that a _Meta Object_ always contain an
-`href`.
+If an `href` member exists, Ion parsers MUST identify the Meta Object as an Ion Link in addition to any other Ion Object identified by the Meta Object.
 
 Note that in the common case, target IRIs and context IRIs will also be
 {uri-URIs}[URI]s (RFC 3986), because many protocols (such as HTTP) do not support
 dereferencing IRIs.  In serializations that do not support IRIs, IRIs will be
-converted to URIs according to {uri-IRIs-3-1}[RFC 3987, Section 3.1].
+converted to URIs according to {uri-IRIs-3-1}[RFC 3987 Section 3.1].
 
-==== `mediaType`
+Use of this member is OPTIONAL.
 
-A resource's https://tools.ietf.org/html/rfc6838[Media Type].  Use of this
-Field is OPTIONAL.
+=== `mediaType`
 
-==== `rel`
+The `mediaType` member identifies the resource's https://tools.ietf.org/html/rfc6838[Media Type].  The `mediaType` value is a case-insensitive string as defined by {uri-mediaType-namingReqs}[RFC 6838 Section 4.2]. Use of this member is OPTIONAL.
```

Comment:
If I could add one though to the mix, it is that in some formats, you have the ability to communicate two different kinds of media types. One is for telling the client what format the server accepts, and another tells the client what format the server will respond with. Just mentioning in case you see a future collision happening.